packages/gateway/src/features/fhir/constants.ts

```diff
@@ -34,3 +34,4 @@ export const VIEWED_EXTENSION_URL = `${OPENCRVS_SPECIFICATION_URL}extension/regV
 
 export const BIRTH_REG_NO = 'birth-registration-number'
 export const DEATH_REG_NO = 'death-registration-number'
+export const METRICS_URL = process.env.METRICS_URL || 'http://localhost:1050'
```

Comment:
@favour234 this can be deleted. There is already a constant here which can be used:  https://github.com/opencrvs/opencrvs-core/blob/0499e054ecfa8d327e081955d649a8bc68241b3f/packages/gateway/src/constants.ts#L23